scripts/setup.cfg

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[metadata]
+name = rh-primaza-control
+version = 0.1.0
+author = Martin Mulholland
+author_email = mmulholl@redhat.com
+description = Primaza control
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/primaza/primazactl
+project_urls =
+    Bug Tracker = https://github.com/primaza/primazactl/issues
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: Apache Software License
+    Operating System :: OS Independent
+
+[options]
+package_dir =
+    = src
+packages = find:
+python_requires = >=3.6
```

Comment:
We're claiming compatibility with python 3.6 & greater, but in CI we're testing 3.10.  Is that intentional?  We might run into problems with this discrepancy.

Reply:
Fixed